airbyte-integrations/connectors/source-github/source_github/spec.json

```diff
@@ -88,6 +88,14 @@
       }
     }
   },
+  "api_url": {
+    "type": "string",
+    "examples": ["api.github.com"],
+    "title": "API URL",
+    "default": "api.github.com",
+    "description": "Please enter your self-hosted URL from Github.",
+    "order": 1
+  },
```

Comment:
This probably should have a format/validation so that folks know not to include the "https" because you are adding that in code

Reply:
Some of the other connectors use regular expressions for this check

Reply:
Included Regex to exclude strings with http or https